src/Features/Core/Portable/ExternalAccess/Pythia/Api/PythiaCompletionProviderBase.cs

```diff
@@ -43,7 +43,7 @@ internal abstract class PythiaCompletionProviderBase : CommonCompletionProvider,
             ImmutableDictionary<string, string>? properties = null,
             ImmutableArray<string> tags = default)
             => SymbolCompletionItem.CreateWithSymbolId(displayText, displayTextSuffix: null, symbols, rules, contextPosition, sortText, insertionText,
-                filterText, displayTextPrefix: null, inlineDescription: null, glyph: null, supportedPlatforms, properties, tags);
+                filterText, displayTextPrefix: null, inlineDescription: null, glyph: null, supportedPlatforms, properties != null ? properties.AsImmutable() : default, tags);
```

Comment:
nit: you've done thsi pattern so many times.  i wonder if we have an extension for it (or if it woudl be good to create one).

Reply:
found AsImmutableOrNull

Reply:
w00t